pkg/log/zap/zap_test.go

```diff
@@ -296,3 +299,135 @@ var _ = Describe("Zap logger setup", func() {
 		})
 	})
 })
+
+var _ = Describe("Zap log level flag options setup", func() {
+	var (
+		fromFlags Options
+		fs        flag.FlagSet
+	)
+
+	BeforeEach(func() {
+		fromFlags = Options{}
+		fs = *flag.NewFlagSet(os.Args[0], flag.ExitOnError) //flags are now reset
+	})
+
+	Context("with  zap-log-level options provided", func() {
+		It("Should set info/debug zap-log-level flags.", func() {
+			args := []string{"--zap-log-level=debug"}
+			fromFlags.BindFlags(&fs)
+			if err := fs.Parse(args); err != nil {
+				Expect(err).ToNot(HaveOccurred())
+			}
```

Comment:
Can't this just be an assertion on the error without checking `err!=nil`?
```suggestion
			err := fs.Parse(args)
			Expect(err).ToNot(HaveOccurred())
```